src/components/02-buttons/buttons.njk

```diff
@@ -1,6 +1,21 @@
-
 <button class="{{ classes }}">Default</button>
 {% if (category != 'disabled') and (category != 'big') %}
-<button class="{{ classes }} usa-button-active">Active</button>
-<button class="{{ classes }} usa-button-hover">Hover</button>
+  <button class="{{ classes }} usa-button-hover">Hover</button>
+  <button class="{{ classes }} usa-button-active">Active</button>
+  <button class="{{ classes }} usa-button-focus">Focus</button>
+  {% if (category == 'secondary') %}
```

Comment:
I think we're supposed to use `{%-` here to remove the whitespace? I might have that wrong though, I haven't done it yet on my PR.